pkg/kubelet/apis/well_known_labels.go

```diff
@@ -19,63 +19,41 @@ package apis
 import (
 	"strings"
 
+	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 const (
-	LabelHostname           = "kubernetes.io/hostname"
-	LabelZoneFailureDomain  = "failure-domain.beta.kubernetes.io/zone"
-	LabelMultiZoneDelimiter = "__"
-	LabelZoneRegion         = "failure-domain.beta.kubernetes.io/region"
-
-	LabelInstanceType = "beta.kubernetes.io/instance-type"
-
-	LabelOS   = "kubernetes.io/os"
-	LabelArch = "kubernetes.io/arch"
-	// The OS/Arch labels are promoted to GA in 1.14. kubelet applies both beta
-	// and GA labels to ensure backward compatibility.
-	// TODO: stop applying the beta OS/Arch labels in Kubernetes 1.17.
-	LegacyLabelOS   = "beta.kubernetes.io/os"
-	LegacyLabelArch = "beta.kubernetes.io/arch"
-
 	// GA versions of the legacy beta labels.
 	// TODO: update kubelet and controllers to set both beta and GA labels, then export these constants
 	labelZoneFailureDomainGA = "failure-domain.kubernetes.io/zone"
```

Comment:
why not these?

Reply:
Likely more context here https://github.com/kubernetes/kubernetes/pull/68267, but the idea was to prevent usage of these labels until kubelet/controllers were applying these labels (in a backwards compatible way). I don't see harm in exporting them now as long as there is a follow up to apply the GA labels where needed (a bunch of cloud specific controllers need this update). @liggitt may have more context here.

Reply:
There probably needs to be a KEP for how we're going to deprecate the beta labels going forward though?

Reply:
> There probably needs to be a KEP for how we're going to deprecate the beta labels going forward though?

Yup. See https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190130-node-os-arch-labels.md as a model.

Reply:
Thanks @liggitt, I can follow up to write a KEP on this for zone/region, instance types and other cloud-specific ones that need to be addressed. Will leave it up to @thockin and @dims if they want to keep these here or until after the KEP is approved.

Reply:
i feel this PR is just for code re-organization ( moving things around ). yes +1 to a follow up KEP for other things